trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.js

```diff
@@ -43,4 +44,28 @@ describe(page, () => {
     wrapper.expectText(tabSelector).toRead(tabName);
     wrapper.expectElement(tabSelector).toHaveAttribute('href', href);
   });
+
+  it('should render the premium payments tab with headings, text, table and buttons', () => {
+    const premiumPaymentsTabSelector = 'div#premium-payments';
+
+    wrapper.expectElement(premiumPaymentsTabSelector).toExist();
+
+    wrapper.expectText(`${premiumPaymentsTabSelector} h2[data-cy="premium-payments-heading"]`).toRead('Premium payments');
+    wrapper
+      .expectText(`${premiumPaymentsTabSelector} p`)
+      .toMatch(/Enter received payments against reported fees by selecting them and then selecting the 'Add a payment' button./);
+    wrapper
+      .expectText(`${premiumPaymentsTabSelector} p`)
+      .toMatch(
+        /When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when you select the 'Generate keying data' button./,
+      );
+
+    wrapper.expectElement(`${premiumPaymentsTabSelector} form[data-cy="premium-payments-form"]`).toExist();
+
+    wrapper.expectElement(`${premiumPaymentsTabSelector} div.govuk-button-group`).toExist();
+    wrapper.expectElement(`${premiumPaymentsTabSelector} button[data-cy="add-a-payment-button"]`).toExist();
+    wrapper.expectElement(`${premiumPaymentsTabSelector} button[data-cy="generate-keying-data-button"]`).toExist();
```

Comment:
Here can we check for buttons with the correct text rather than just the `data-cy` attribute?